product_docs/docs/biganimal/release/reference/cli.mdx

```diff
@@ -189,6 +189,25 @@ biganimal create-cluster --clusterConfigFile ./config_file.yaml
 !!! Tip
 You can turn off the confirmation step with the `biganimal disable-confirm` command.
 !!!
+### Get cluster connection information
+
+To use your BigAnimal cluster, you first need to get your cluster's connection information. To get your cluster's connection information, use the `show-cluster-connection` command:
+
+```
+$ biganimal show-cluster-connection --name> my-biganimal-cluster --provider azure --region eastus
```

Comment:
"--name>", here the ">" is typo , it shouldn't be here.

Reply:
Thanks. Fixed.